src/MurmurHash.Net/MurmurHash3.cs

```diff
@@ -5,7 +5,13 @@ namespace MurmurHash.Net
 {
     public class MurmurHash3
     {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static uint Hash32(Span<byte> bytes, uint seed)
+        {
```

Comment:
If you don't care about binary backward comparability, this can be a much simpler change. The type of `bytes` just needs to be changed from `Span<byte>` to `ReadOnlySpan<byte>`. Let me know if you're OK with the simpler version and I can revise.

Reply:
Great! Let's change `Span` to `ReadOnlySpan`. It is not necessary to be able to modify data inside the method.

Reply:
Revised to just be the single change from `Span` to `ReadOnlySpan`.